tests/test-delta.sh

```diff
@@ -156,6 +156,13 @@ ${CMD_PREFIX} ostree --repo=repo2 ls ${newrev} >/dev/null
 
 echo 'ok pull delta'
 
+touch test-xattrs
+if ! setfattr -n user.testvalue -v somevalue test-xattrs; then
+    echo "ok apply offline # SKIP bare-user repository requires xattr support"
+    echo "ok apply offline inline # SKIP bare-user repository requires xattr support"
+    exit 0
```

Comment:
Oh, I see.  Hm, yeah that's probably going to be a maintenance pain over time.  I think I'd vote for skipping the whole file, assume we get test coverage from systems that aren't tmpfs for a while.